packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/migrate.ts

```diff
@@ -0,0 +1,21 @@
+import { CONTRACT_LIST } from '../../../lib/contracts'
+import { MigrateContract, MigrationMessageMaker } from '../migration'
+
+// TODO: Pending of migrate function on the contract. Serve this as an example of implementation
+type OCR2MigrateMessage = any
+
+const makeMigrationMessage: MigrationMessageMaker<OCR2MigrateMessage> = (flags, args) => {
+  return {}
+}
+
+// export default makeMigrationCommand(CONTRACT_LIST.OCR_2, makeMigrateMessage)
+
+export default class OCRMigrateCommand extends MigrateContract<OCR2MigrateMessage> {
```

Comment:
First, notice the `OCR` vs `OCR2` prefix.
Second, I don't think we should namespace like this as we can always `import {A as B}` It's already clear this command is in OCR2 context as it's located under `/contracts/ocr2/`.